.idea/php-test-framework.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="PhpTestFrameworkVersionCache">
+    <tools_cache>
+      <tool tool_name="PHPUnit">
+        <cache>
+          <versions>
+            <info id="Local" version="7.0.2" />
+          </versions>
+        </cache>
+      </tool>
+    </tools_cache>
+  </component>
+</project>
```

Comment:
I'm not 100% sure if this fime should be committed, it seems to contain configuration specific to your computer. Maybe we should ignore the file for now?

Reply:
I was actually thinking the same, but I guess it's ok.